pagopa/io-app
# chore: [IOPID-94] Add Lollipop Playground

## ⚠️ Depends on https://github.com/pagopa/io-functions-lollipop/pull/38, https://github.com/pagopa/io-backend/pull/1006, https://github.com/pagopa/io-dev-api-server/pull/233 ⚠️ ##

## Short description
This PR add a Lollipop Playground page to call the first LC for debugging purpose.

## List of changes proposed in this pull request
- [package.json](https://github.com/pagopa/io-app/pull/4545/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519): add OpenApi specs and client generation.
- [ts/features/lollipop/api/backend.ts](https://github.com/pagopa/io-app/pull/4545/files#diff-f74e1d5ec082224b0319d628738f4276acd718325ec8adf1b024edcf831d054a): Adds the lollipop backend server.
- [ts/features/lollipop/playgrounds/LollipopPlaygroundContent.tsx](https://github.com/pagopa/io-app/pull/4545/files#diff-d1d3dd0d4f65dbaffcc58b5b41f71abc080812ff110a7195addc8c960392163b):
- [ts/features/lollipop/playgrounds/LollipopPlayground.tsx](https://github.com/pagopa/io-app/pull/4545/files#diff-7177b8ec63fef44a3ace9d084515adeefd1a8979d89f9cc3030989173797799d): Adds the lollipop Playground Screen.
- [ts/navigation/ProfileNavigator.tsx](https://github.com/pagopa/io-app/pull/4545/files#diff-fde597880587bbd6b7df5084df5a2bb4020db9a4d14f00b29967f3faf58ad7a1):
- [ts/navigation/params/ProfileParamsList.ts](https://github.com/pagopa/io-app/pull/4545/files#diff-f3f50220ac440ed85361a813ef5aff2d66dc324b38c8d47f9de541651342fb41):
- [ts/navigation/routes.ts](https://github.com/pagopa/io-app/pull/4545/files#diff-950467b6141738ddf5a35aa966af7ea9731f4f70825f4fcf2e3d62f33f7ac547):
- [ts/screens/profile/ProfileMainScreen.tsx](https://github.com/pagopa/io-app/pull/4545/files#diff-de3c1a64bcc8fdca4c8dbc5d62dba36c8c5ded27255052ecc6526e614b1e8f5b): Manages the navigation to the lollipop playground from the profile page in debug mode.

## How to test
Run the app and go to the new lollipop playground screen. Write something in the text field and tap "Sign". You should see a "success" alert showing the current server assertion ref. Test it by pointing to production and also by using the dev server.
